clang/test/Misc/r600.languageOptsOpenCL.cl

```diff
@@ -146,3 +146,8 @@
 #pragma OPENCL EXTENSION cl_khr_subgroups: enable
 // expected-warning@-1{{unsupported OpenCL extension 'cl_khr_subgroups' - ignoring}}
 
+#ifdef __opencl_c_generic_address_space
+#error "Incorrect __opencl_c_generic_address_space define"
```

Comment:
I personally think the error message is confusing…

Reply:
Yeah I do too, but I was trying to go with the style of the other checks in this test. Or is it particularly this generic address space one, and the others are okay?

Reply:
It's not a real error so it doesn't really matter